docs/en/user_guides/train_and_test.md

```diff
@@ -1,5 +1,22 @@
 # Training and Test
```

Comment:
Why do you use `Training and Test`, this is so weird..

Reply:
I think it is because this doc uses the corresponding [MMClassification doc](https://github.com/open-mmlab/mmclassification/blob/1.x/docs/en/user_guides/train_test.md) as a reference. 

Do you suggest changing 'Test' to 'Testing'?

Reply:
Personally I prefer `Train & Test` like MMDetection

:D

![image](https://user-images.githubusercontent.com/13503330/188591060-56d3e503-4773-4568-9fa5-2ac9f3bc3ee7.png)